Wiki/de/index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Gridcoin Wiki
+layout: wiki-de
+---
+
+## Gridcoin Einführung
+
+Gridcoin (Ticker: GRC) ist eine dezentrale, mathematische,
+open source Kryptowährung (digitaler Vermögenswert). 
+Gridcoin ist, neben seiner Energieeffizienz, auch die
+erste und derzeit einzige Kryptowährung, die Einzelpersonen
+für wissenschafftliche Beiträge belohnt und peer-to-peer 
+Transaktionen kryptographisch vollzieht, ohne dabei auf 
+eine zentrale Instanz zur Verteilung der Belohnungen angewiesen 
+zu sein. Gridcoin ist das erste Blockchain Protokoll welches 
+einen funktionierenden Algorithmus zum Nachweis und Belohnung von 
+geleisteter [BOINC](https://en.wikipedia.org/wiki/Berkeley_Open_Infrastructure_for_Network_Computing)
+(Berkeley Open Infrastructure for Network Computing) Rechenarbeit,
+wobei es sich um quasi nahezu jeglichen Computerprozess handeln 
+kann, liefert.
```

Comment:
I would rephrase the sentence. 
```suggestion
Gridcoin ist dank der Verwendung des Proof of Stake Konsensus-Mechanismus sehr energieeffizient. Gridcoin ist die
erste und derzeit einzige Kryptowährung, die Einzelpersonen
dafür Belohnt, dass sie ihre Rechenleistung für wissenschafftliche Forschung zur Verfügung stellen. Dabei werden alle Transaktionen kryptographisch sicher und ohne eine zentrale Instanz abgewickelt.

Nutzer, die ihre Rechenleistung [BOINC Projekten](https://en.wikipedia.org/wiki/Berkeley_Open_Infrastructure_for_Network_Computing)
(Berkeley Open Infrastructure for Network Computing) zur Verfügung stellen, können damit Gridcoins verdienen.
```

Reply:
Yes, PoS is more energy efficient, because a new hash can only be generated every 16 seconds (or 8?).

Reply:
I think we talk about different things. 
In my Opinion, an energy efficient PC is shut down when not used. But, when this PC is crunching during idle times is consumes much more energy than in idle mode!! 
But the BIG difference is, when a PC is dedicated to mining a Coin, he will consume much more energy, just that is his only task. 
So at least, it isn't "more" energy efficient (the german meaning). It is only.... you do something good when your PC is on power and YOU do nothing with it. 

for explanation: The german energy efficient (energieffiezient) meaning is a very narrow thing, like LED is much more energy efficient than a light bulb. 
But in this case, your PC is running at 100% while you do nothing but you can earn a little bit money and help some science guys. (GRC) vs. your PC is running at 100% just for mining money (ex. BTC). 

Sorry for my long explanation. The **text isn't wrong** and in the last 20 years, I recommended BOINC everybody who run their computers 24/7. So I am on your side, I just don't like the word energy efficient in this meaning.

Reply:
The idea (at least from what its trying to convey from the English version) is that proof of stake itself isn't what's using up energy. The system of proof of stake doesn't use up much energy and so most of the computations involved in Gridcoin are on BOINC (not a part of proof of stake) meaning that they're useful. This means that overall there's less wasted computation in Gridcoin. 

Maybe this could be phrased differently in German to clarify?

Reply:
@lucky2d thanks for the clarification. Now I see what you mean.
I will change the text a bit.

Reply:
@lucky2d can you check over the edited suggestion?

Here is their edited suggestion for reference
```suggestion
Das Erstellen von neuen Blöcken auf der Gridcoin Blockchain ist dank der Verwendung des Proof of Stake Konsensus-Mechanismus sehr energieeffizient. Gridcoin ist die
erste und derzeit einzige Kryptowährung, die Einzelpersonen
dafür Belohnt, dass sie ihre Rechenleistung für wissenschafftliche Forschung zur Verfügung stellen. Dabei werden alle Transaktionen kryptographisch sicher und ohne eine zentrale Instanz abgewickelt.
Nutzer, die ihre Rechenleistung [BOINC Projekten](https://en.wikipedia.org/wiki/Berkeley_Open_Infrastructure_for_Network_Computing)
(Berkeley Open Infrastructure for Network Computing) zur Verfügung stellen, können damit Gridcoins verdienen
```

Reply:
Thank you @RoboticMind. Now, it sounds very good. Just only one typo in line 12. The word "Belohnt" is a verb and therefore it is writte in small letters "belohnt"